service/stop.go

```diff
@@ -11,11 +11,16 @@ func (svc *service) StopApp() {
 	if svc.appCancelFn != nil {
 		logger.Logger.Info("Stopping app...")
 		svc.appCancelFn()
+		svc.swapsService.StopAutoSwaps()
```

Comment:
What happens if a swap is in progress when we shut down? there's no issue, right?

Reply:
Added a commit to make sure we also stop the boltz websocket, should be good now